charts/kyverno/templates/hooks/clusterrole.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "kyverno.pre-upgrade-job.name" . }}
+  annotations:
+    "helm.sh/hook": pre-upgrade
+    "helm.sh/hook-weight": "-10"
+rules:
+- apiGroups: [""]
+  resources: [configmaps]
+  verbs: [get, list, patch, create, delete]
```

Comment:
I don't see any listing or manipulation of configmaps, so this capability isn't needed.  You shouldn't need this just to have a configmap as a volume mount in the pod.

Reply:
Implemented all review comments and tested. Thank you.